images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -266,15 +266,8 @@ function Get-VSExtensionVersion
         [string] $packageName
     )
 
-    $instanceFolders = Get-ChildItem -Path "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances"
```

Comment:
Could you please move `Get-VSExtensionVersion` function to `VisualStudioHelpers.ps1`?